### ISSUE TITLE
undefined message is always in `Ready` state

### ISSUE THREAD

**Issue by yamii** [derickbailey/rabbus#7]
I am using request/respond library accidentally I've passed a `undefined` message to the request. The request stayed in `Ready` state and error is not emitted.

I'm thinking of trapping undefined message but is there a better way to handle this scenario?

**Comment by rhyslbw** [comment]
Can probably just close this right?

**Comment by mxriverlynn** [comment]
i never got around to testing this scenario.... it would be good to try it and see what happens. if there's a design error or something that we can do to make this easier to handle, then we should

also - sorry i took so long to get to this @yamii ... i should have looked into this a loooong time ago

**Comment by yamii** [comment]
@derickbailey yeah. this was looong time ago ( anyways this is a least prio ). But I do have a workaround and I haven't tested it again with the latest updates in rabbus. I'm thinking to have an `error` emitted when message is `undefined`
